draft-ietf-privacypass-architecture.md

```diff
@@ -410,15 +423,184 @@ guidelines on changes to the parametrization in {{parametrization}}.
 Any extension to the Privacy Pass protocol MUST adhere to the guidelines
 specified in {{issuer-role}} for managing Issuer public key data.
 
-# Privacy considerations {#privacy}
+# Deployment Considerations {#deployment}
+
+Client uses Privacy Pass to separate attestation context and redemption
+context. Linking or combining these contexts can reveal sensitive information
+about the Client, including their identity or browsing history. Depending on the
+deployment model, separating these contexts can take different forms.
+The Origin, Attester, and Issuer portrayed in {{fig-overview}} can be instantiated
+and deployed in a number of different ways. This section covers some expected
+deployment models and their corresponding security and privacy considerations.
+The discussion below assumes non-collusion between entities when operated by
+separate parties.
```

Comment:
Should we mention at all how the client knows which entities are separate? If that's out of scope, can we say so or make a reference?

Reply:
Yup, that would be good to add.